addon/serializers/github-repository.js

```diff
@@ -7,6 +7,13 @@ export default GithubSerializer.extend({
       fullName: hash.full_name,
       name: hash.name,
       description: hash.description,
+      url: hash.html_url,
+      language: hash.language,
+      isFork: hash.fork,
+      isPrivate: hash.private,
```

Comment:
- url: there actually is a `url` property besides `html_url` so I definitely want to keep that to `htmlUrl`
- isFork/isPrivate: totally agree with your logic and I agree with your point about the abstraction layer, I'll give this one some more thought on what direction I want to go...